Services/Calendar/classes/class.ilCalendarBlockGUI.php

```diff
@@ -268,25 +268,6 @@ public function executeCommand(): string
         return '';
     }
 
-    public function fillDataSection(): void
-    {
-        if ($this->display_mode != "mmon") {
-            $this->setRowTemplate("tpl.pd_event_list.html", "Services/Calendar");
```

Comment:
Please also remove the template tpl.pd_event_list.html It is not used elsewhere.